src/hooks/mongodb4x.test.js

```diff
@@ -0,0 +1,86 @@
+import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
+import { getMockedMongoClient, wrapMongoCollection } from '../../testUtils/mongo4xMocker';
```

Comment:
I positively _hate_ mocks in instrumentation tests. They hide infinite bugs and regressions. Can't we rather run the tests against the actual package and maybe, just maybe, a sprinkling of https://github.com/testcontainers/testcontainers-node?

Reply:
for the moment they're staying, to be discussed further